source/dub/packagemanager.d

```diff
@@ -203,9 +203,19 @@ class PackageManager {
 	bool isManagedPackage(Package pack)
 	const {
 		auto ppath = pack.basePackage.path;
+		return isManagedPath(ppath);
+	}
+
+	/** Determines if a specifc path is within a DUB managed package folder.
+
+		By default, managed folders are "~/.dub/packages" and
+		"/var/lib/dub/packages".
```

Comment:
Nice, didn't knew there was a system-wide installation path.
This is useful for caching common dependencies, why is it deprecated?